modules/programs/kubecolor.nix

```
@@ -89,6 +92,12 @@ in {
 
     home.shellAliases = lib.mkIf (cfg.enableAlias && (cfg.package != null)) {
       kubectl = lib.getExe cfg.package;
+      oc = "env KUBECTL_COMMAND=${lib.getExe pkgs.openshift} ${
+          lib.getExe cfg.package
+        }";
```

```suggestion
      oc = lib.mkIf (lib.elem pkgs.openshift config.home-manager.users.ajgon.home.packages || lib.elem pkgs.openshift config.environment.systemPackages) "env KUBECTL_COMMAND=${lib.getExe pkgs.openshift} ${
          lib.getExe cfg.package
        }";
```

I would add this alias only if user has openshift installed.